components/sup-protocol/protocols/types.proto

```diff
@@ -55,7 +55,7 @@ message PackageIdent {
 }
 
 message ProcessStatus {
-  optional int64 elapsed = 1;
+  optional uint64 elapsed = 1;
```

Comment:
Do we have to worry about breaking protocol compatability here?

Reply:
I mentioned this in the commit message, but yeah, I think it should be fine:

https://developers.google.com/protocol-buffers/docs/proto#updating